docs/api-reference/carto/styles.md

```diff
@@ -0,0 +1,129 @@
+# Styles
+
+`CARTO styles` are a set of scale functions to easily assign a color to a feature.
+
+Transform data values into visual variables and take the advantage of [CARTOColors](https://carto.com/carto-colors/) custom color schemes.
+
+## Color Bins
+
+`colorBins` maps continuous numeric input to discrete values defined by the range (`colors`).
+
+* Quantitative scales have a continuous domain.
+
```

Comment:
Need to explain each parameter, here.  The structure should be:

- Description
- Parameters
- Usage example